packages/new-deploy/src/bridge/BridgeContracts.ts

```diff
@@ -146,14 +149,25 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
     const factory = new UpgradeBeaconProxy__factory(
       this.context.getDeployer(name),
     );
-    const prx = await factory.deploy(beacon, initData, this.overrides);
+
+    const constructorArguments = [beacon, initData];
+    const prx = await factory.deploy(
+      beacon, //constructorArguments[0],
```

Comment:
that's me being lazy with types. `constructorArguments[]` contains `ethers.BytesLike` which is less specific than `beacon: string`. so casting from an element of the args array to string is not safe from the typesystem's perspective. rather than clutter up the logic with casts, I decided to leave a not for future james